govc/cluster/module/rm.go

```diff
@@ -36,30 +40,53 @@ func init() {
 func (cmd *rm) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
 	cmd.ClientFlag.Register(ctx, f)
+
+	f.StringVar(&cmd.file, "file", "", "File containing a list of module ids to delete")
 }
 
 func (cmd *rm) Usage() string {
-	return "ID"
+	return "[-file=] [ID]"
 }
 
 func (cmd *rm) Description() string {
 	return `Delete cluster module ID.
 
+When -file is specified, it reads and deletes all modules contained in the file.
+
 Examples:
-  govc cluster.module.rm module_id`
+  govc cluster.module.rm module_id
+  govc cluster.module.rm -file modules.txt`
 }
 
 func (cmd *rm) Run(ctx context.Context, f *flag.FlagSet) error {
-	if f.NArg() != 1 {
+	if f.NArg() != 1 && cmd.file == "" {
```

Comment:
This doesn't look quite right: does this return ErrHelp for eg `-file=foo bar1 bar2`?

Reply:
Ah yeah, I should not rush a PR shortly before calling it a day ... 👍